ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -75,7 +75,7 @@ class TabBarViewDemoController: DemoController {
 
         if showsItemTitles {
             updatedTabBarView.items = [
-                TabBarItem(title: "Home", image: UIImage(named: "Home_24")!, selectedImage: UIImage(named: "Home_Selected_24")!),
+                homeItem,
```

Comment:
Can you explain the root cause of the issue and how the changes in code fix the issue? (in the description of the PR)

Reply:
I have put the description.